keyboards/space_space/rev2/rev2.c

```diff
@@ -0,0 +1,17 @@
+
+/* Copyright 2021 qpockets
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "rev2.h"
```

Comment:
This way, the encoder function is set by default but can be overridden by users, and will still work for the configurator.
```suggestion
#include "rev2.h"

bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (index == 1) { /* left encoder*/
                if (clockwise){
                    tap_code(KC_WH_U);
                } else {
                    tap_code(KC_WH_D);
                }
    } else if (index == 0) { /* right encoder */
                if (clockwise){
                    tap_code(KC_VOLU);
                } else {
                    tap_code(KC_VOLD);
        }
    }
    return true;
}
```